src/applications/vre/28-8832/containers/IntroductionPage.jsx

```diff
@@ -8,153 +8,172 @@ import { focusElement } from 'platform/utilities/ui';
 import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
 import SaveInProgressIntro from 'platform/forms/save-in-progress/SaveInProgressIntro';
 import { getEligiblePages } from 'platform/forms-system/src/js/routing';
+import { setData } from 'platform/forms-system/src/js/actions';
 import recordEvent from 'platform/monitoring/record-event';
 import {
   CHAPTER_31_ROOT_URL,
   WIZARD_STATUS,
   PCPG_ROOT_URL,
 } from '../constants';
 
-class IntroductionPage extends React.Component {
-  componentDidMount() {
+const IntroductionPage = props => {
+  // focus on element
+  const [pageList, setPageList] = useState(props.route.pageList);
+  useEffect(() => {
     focusElement('.va-nav-breadcrumbs-list');
```

Comment:
I think we need focus to be on the `h1`, and the content needs to change once the user has completed the wizard - see https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/disability-benefits/996/containers/IntroductionPage.jsx#L77-L86 - please talk to Trevor about this.

Reply:
Also, this should cause a new render so I think the `userEffect` should take care of moving the focus, but it would be good to verify.

Reply:
I think adding the focus in the wizard container will address this issue as well.